webstorage/defineProperty.window.js

```diff
@@ -0,0 +1,57 @@
+["localStorage", "sessionStorage"].forEach(function(name) {
+    [9, "x"].forEach(function(key) {
```

Comment:
Test with a Symbol as well? That's out of scoped for "named properties" though. Separate test?